admin.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Bens Admin Page</title>
+    <link rel='stylesheet' type='text/css' href='css/normalise.css'>
+    <link rel='stylesheet' type='text/css' href='css/admin.css'>
+</head>
+
+<body>
+    <header>
+        <h1>Admin Page</h1>
+        <h2>Welcome Ben</h2>
+        <h3>About Me</h3>
+    </header>
+    <div class="forms">
+        <div class="container">
+            <h4>Add information</h4><form method="post">
```

Comment:
Add the <form> tag to a new line... Also the form needs to have a destination...

Reply:
Will add destination when it has been created